src/components/settings/SettingsDrawer.jsx

```diff
@@ -121,11 +122,11 @@ export function SettingsDrawer({ isOpen, onClose }) {
                             </a>
                             <SettingsOption Icon={FaShareAlt} onClick={handleShare} text={i18n.t("share_app")} />
                             <a
-                                href="https://gitlab.com/opensourceuom/myUoM"
+                                href="https://github.com/open-source-uom/myuom"
                                 target="_blank"
                                 rel="noopener noreferrer"
                             >
-                                <SettingsOption Icon={FaGitlab} onClick={onClose} text={i18n.t("gitlab_contribute")} />
+                                <SettingsOption Icon={FaGithub} onClick={onClose} text={i18n.t("github_contribute")} />
```

Comment:
You've correctly updated the i18n variable name from `gitlab_contribute` to `github_contribute`, but it needs to be updated in the `settings_page.js` files as well in order for it to work. For both `el` and `en` locales.

Reply:
@NikolasPpd oh my bad. Thanks again